build/update-olm.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import yaml
+
+
+_ANNOTATIONS = {
+    'categories': 'OpenShift Optional',
+    'containerImage': 'quay.io/kubevirt/node-maintenance-operator',
+    'repository': 'https://github.com/kubevirt/node-maintenance-operator',
+    'description': \
+        'Node-maintenance-operator maintains nodes in cluster',
+}
+_DESCRIPTION = "Node maintenance operator"
+_NAMESPACE = 'node-maintenance-operator'
+_SPEC = {
+    'description': _DESCRIPTION,
+    'provider': {
+        'name': 'KubeVirt project'
+    },
+    'maintainers': [{
+        'name': 'KubeVirt project',
+        'email': 'kubevirt-dev@googlegroups.com',
+    }],
+    'keywords': [
+        'KubeVirt', 'Virtualization', 'Node-maintenance'
+    ],
+    'links': [{
+        'name': 'KubeVirt',
+        'url': 'https://kubevirt.io',
+    }, {
+        'name': 'Source Code',
+        'url': 'https://github.com/kubevirt/node-maintenance-operator'
+    }],
+    'labels': {
+        'alm-owner-kubevirt': 'nodemaintenanceoperator',
+        'operated-by': 'nodemaintenanceoperator',
+    },
+    'selector': {
+        'matchLabels': {
+            'alm-owner-kubevirt': 'nodemaintenanceoperator',
+            'operated-by': 'nodemaintenanceoperator',
+        },
+    },
+}
+
+_CRD_INFOS = {
+    'nodemaintenances.kubevirt.io': {
+        'displayName': 'KubeVirt node maintenance',
+        'description': \
+                'Represents a deployment of node maintenance',
```

Comment:
Just a suggestion, these don't really explain what the creation of a nodemaintenance CR will do.